### PR TITLE
chore: add .NET release steps

### PR DESCRIPTION
**Pull request by josecorella**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: DynamoDbEncryption/codebuild/release-staging.yml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+phases:
+  install:
+    runtime-versions:
+      dotnet: 6.0
+      python: 3.x
+    commands:
+      - cd ..
+      # Get Dafny
+      ## TODO FIX ME
```
Comment (ajewellamz) [inline DynamoDbEncryption/codebuild/release-staging.yml:11]: To do what? Fix what?

File: DynamoDbEncryption/codebuild/release-prod.yml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+phases:
+  install:
+    runtime-versions:
+      dotnet: 6.0
+      python: 3.x
+    commands:
+      - cd ..
+      # Get Dafny
+      ## TODO FIX ME
```
Comment (ajewellamz) [inline DynamoDbEncryption/codebuild/release-prod.yml:11]: TODO?

File: DynamoDbEncryption/codebuild/release-staging.yml [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+      # This step assumes signing has already happened and we just need to retrieve
+      # the signed artifacts
+      - export ASSEMBLY_NAME="AWS.Cryptography.DbEncryptionSDK.DynamoDb.dll"
+      - export BASE=../codebuild/net
+      - python $BASE/retrieve_signed_assembly.py --target net6.0 --unique-id $UNIQUE_ID
+      - python $BASE/retrieve_signed_assembly.py --target net48 --unique-id $UNIQUE_ID
+  build:
+    commands:
+      - mkdir build
+      - >-
```
Comment (ajewellamz) [inline DynamoDbEncryption/codebuild/release-staging.yml:36]: Is this command '>-' correct?

Comment (josecorella) [reply DynamoDbEncryption/codebuild/release-staging.yml:35]: yup!

**Review by ajewellamz — APPROVED** [review]
LGTM

**Review by ajewellamz — DISMISSED** [review]
LGTM

**Review by ajewellamz — APPROVED** [review]

**Pull request merged**